transform/src/test/resources/transformations/message-trait/webapi.yaml

```diff
@@ -0,0 +1,36 @@
+#%WebAPI Spec 1.0
+unitType: Document
+$id: file://transform/src/test/resources/transformations/message-trait/api.yaml#/rootAsset
+declares: []
```

Comment:
Why is the declares array empty? In the JSON-LD it's not empty. Is this a bug in AML emission?